sdk/tools/gcc_plugin_seh/main.cpp

```diff
@@ -0,0 +1,175 @@
+/* This file is in the public domain and has no copyright */
```

Comment:
Me. And public domain is GPL compatible. And I want people to do whatever they want with this particular piece of code.

Reply:
Public domain is a vague term in some jurisdictions. It might be better to release it under CC0, which is, unlike "public domain", a clearly defined license (functionally equivalent to the intended public domain placement).

Reply:
0BSD it is then.